config/initializers/rails_5.rb

```diff
@@ -0,0 +1 @@
+ActiveRecord::ConnectionAdapters::SQLite3Adapter.represent_boolean_as_integer = true
```

Comment:
I had to add an initializer for the frame-renderer to reset all the boolean fields to integers to backport this to DBs that already exist.

I believe this app is used at other sites (at least 1) so it's likely something we should account for.